src/formState.ts

```diff
@@ -476,7 +476,7 @@ function newObjectState<T, P = any>(
         throw new Error(`${key || "formState"} is currently readOnly`);
       }
       getFields(this).forEach((field) => {
-        if (field.key in value && (!field.dirty || !field._focused)) {
+        if (field.key in value) {
```

Comment:
This was the 1st attempt that is removed and we're just doing it down below now.